lib/gds-components/radios/Radios.tsx

```diff
@@ -47,12 +49,16 @@ export const Radios = ({
                 className="govuk-radios__input"
                 type="radio"
                 value={value}
-                id={id}
+                id={value}
```

Comment:
we were using the id from the components props here instead of something unique to the radio option. Because of that clicking on the radio label wasn't selecting the right radio option. The props id is used to navigate to the radios from the errorSummary link so I have assigned that id to the fieldset and an id specific to each option to the radio inputs